src/app/features/landing/components/landing-footer/landing-footer.component.html

```diff
@@ -0,0 +1,40 @@
+<footer class="landing-footer">
+  <div class="footer-grid">
+    <div class="col-12 md:col-2" style="margin-top: -1.5rem">
+      <a (click)="router.navigate(['/pages/landing'], { fragment: 'home' })">
+        <h4 class="footer-logo">Foundation</h4>
```

Comment:
Get these text values from the i18n language json file instead with translation service.

Reply:
I have added localization feature to the app in my latest commit. The localization has following features

1. For the first time visitor detects browser language and translate the app to that language if available.
2. User get an option to choose the languages in the footer.
3. The app gets translated to chosen language and the preference gets saved until the user changes it.